sematic/examples/testing_pipeline/__main__.py

```diff
@@ -154,6 +158,7 @@
     "Includes a function which will exit with the specified code. "
     "If specified without a value, defaults to 0. Defaults to None."
 )
+CUSTOM_RUNNER_RESOURCES_HELP = "Specifies custom resources for the CloudRunner."
```

Comment:
Please mention that a hardcoded token value is used.

Reply:
done!